### ISSUE TITLE
Trove classifiers are incorrect

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7333]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#7333|
|Type|defect|
|Created|2014-05-12 11:22:55Z|
|Branch|https://github.com/twisted/twisted/tree/setup3-same-classifiers-7333|

https://pypi.python.org/pypi/Twisted/14.0.0 still says

```
    Programming Language :: Python :: 2.6
    Programming Language :: Python :: 2.7
    Programming Language :: Python :: 2 :: Only
```

which is kind of unfortunate.

Turns out, we’re still setting the 3.3 classifier in setup3.py which is wrong.  The metadata must be consistent for all versions.

<details><summary>Searchable metadata</summary>

```
trac-id__7333 7333
type__defect defect
reporter__hynek hynek
priority__normal normal
milestone__None None
branch__branches_setup3_same_classifiers_7333 branches/setup3-same-classifiers-7333
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1399893775000000 1399893775000000
changetime__1445217216921098 1445217216921098
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @exarkun|
|-|-|

> ​https://pypi.python.org/pypi/Twisted/14.0.0 still says

As far as I can tell, as `setup.py` is now, this will be fixed in 14.1.0 (check `setup.py --classifiers`).  This was fixed as part of [#6539](https://github.com/twisted/twisted/issues/6539).

> Turns out, we’re still setting the 3.3 classifier in setup3.py which is wrong. The metadata must be consistent for all versions.

I'm not clear on this part though.  I guess you want that classifier deleted from setup3.py?  I don't think it actually makes a difference because we don't do releases with setup3.py.  But deleting it is super easy I guess so ...

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [43050]) Branching to 'setup3-same-classifiers-7333'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @glyph|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_6"></a>|@khorn commented|
|-|-|

Looks good to me.  Merge it.

Shortest review ever?

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @exarkun|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [43059]) Merge setup3-same-classifiers-7333

Author: exarkun (sponsored by Hybrid Logic, Ltd.)
Reviewer: khorn
Fixes: [#7333](https://github.com/twisted/twisted/issues/7333)

Remove the trove classifier override in the Python 3 distutils entrypoint.
This makes the classifiers consistent whether Python 2 or Python 3 is being
used.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_9"></a>|@hawkowl commented|
|-|-|

Ticket retargeted after milestone deleted
